docs/migrating_to_slash_commands.rst

```diff
@@ -0,0 +1,43 @@
+:orphan:
+
+.. _migrating_to_slash_commands:
+
+
+Migrating To Slash Commands
+=============================
+
+Differences
+-------------
+One of the biggest difference's is the deprevation of CTX(Context) in the use of interaction 
+
+**NOTE:** It may take up to an hour for a slash command to register globally so we recommend you put guild_ids[] to limit the amount of guilds for testing 
```

Comment:
```suggestion
.. note::

    It may take up to an hour for a slash command to register globally so we recommend you put guild_ids[] to limit the amount of guilds for testing 
```